### DISCUSSION TITLE
[Feature Request] Return Form to Initial Submitter for Corrections

### DISCUSSION THREAD

**Discussion by GXY2017** [docusealco/docuseal#135] [Ideas]
The function to set multiple submitters for a form is fantastic. 

However, I suggest adding a feature to return the form to the first submitter. This way, fields filled out incorrectly can be modified without requiring all the submitters to refill them. Thank you.

**Comment by omohokcoj** [comment]
@GXY2017 thanks for the feedback - we have this in the roadmap and hoping to have this release later this year.
